SquashMergeRequirements.md

```diff
@@ -7,10 +7,24 @@ When commiting your PR, please make sure to do these steps at Squash/Merge to as
 1. Tag the PR with the type of update: **Bug**, **Enhancement**, or **New-Feature**
 1. Tag the PR with **Azure API for FHIR** if this will release to the managed service
 1. Include a user friendly, 1-2 sentence in the Squash/Merge **description** wrapped at 72 characters
-1. Note if it **addresses a GitHub issue and/or a VSTS item** in the Squash/Merge description with a link (i.e. #1234 or AB#12345)
+1. Note if it **addresses a GitHub issue and/or a VSTS item** in the Squash/Merge description (i.e. #1234 or AB#12345)
 
 Sometimes our changes will create breaking changes or may need a warning associated with them. If this is the case, take these additional steps:
 
 1. **Create a new issue** in GitHub that describes the warning/issue
 1. Tag the PR with **KI-Breaking** or **KI-Warning**
 1. **Reference the new GitHub issue** from the PR description
+
+## Example
+
+Squash/Merge commit heading:
+
+PR title + PR number (maximum 50 characters)
```

Comment:
Just for my clarification, is the PR number also included in the 50 character limit?

Reply:
Good question. I would assume the PR number is included but I am not 100% sure. @brendankowitz would you happen to know how this affects the release notes? I can also defer to @CaitlinV39 when she is back on Monday.

Reply:
I think all characters are included. Lets just see how we can and assess how challenging this is :)

Reply:
Clarified this in 0074627.